src/DataWrangling/EN4/EN4_metadata.jl

```diff
@@ -0,0 +1,197 @@
+using CFTime
```

Comment:
This file is basically all new, so there is no overlap (as far as I can tell) with the ECCO_metadata file

Reply:
yeah, defining new methods for a particular dataset metadata ideally should be the only addition we need.